src/mixins/filing-template-mixin.ts

```diff
@@ -92,7 +92,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     switch (this.getEntityType) {
       case CorpTypeCd.COOP:
         filing.filing.incorporationApplication.cooperative = {
-          cooperativeAssociationType: this.getDefineCompanyStep.cooperativeType
+          cooperativeAssociationType: this.getDefineCompanyStep.cooperativeType,
+          rulesFileKey: this.getRules.docKey,
+          rulesFileName: this.getRules.rulesDoc ? this.getRules.rulesDoc.name : null
         }
         filing.filing.incorporationApplication.rules = this.getRules
         filing.filing.incorporationApplication.memorandum = this.getMemorandum
```

Comment:
We don't need rules and memorandum in incorporationApplication anymore since it will be part of cooperative

Reply:
Doesn't a Cooperative IA schema include an IncorporationApplication block for the contents of the filing?
Could you link the schema?

Reply:
Yes cooperative is inside incorporationApplication. Please have a look into this PR https://github.com/bcgov/business-schemas/pull/69

Reply:
Oh i see, its nested within `incorporationApplication. cooperative`. I got confused by your initial comment, i misunderstood.

Reply:
Argus, please update this code to nest `rules` and `memorandum` under `cooperative` on line 94 above. Or are these obsolete now that you use the file key and name?

Reply:
So how does this work for everyone.  When building the filing, I will grab all properties that need to be saved and push it into the `cooperative` property for the IA filing.  i.e.

```
filing.filing.incorporationApplication.cooperative = {
          cooperativeAssociationType: this.getDefineCompanyStep.cooperativeType,
          rulesFileKey: this.getRules.docKey,
          rulesFileName: this.getRules.rulesDoc ? this.getRules.rulesDoc.name : null,
          rulesFileSize: this.getRules.rulesDoc ? this.getRules.rulesDoc.size : null,
          rulesFileLastModified: this.getRules.rulesDoc ? this.getRules.rulesDoc.lastModified : null,
          rulesConfirmed: this.getRules.rulesConfirmed
        }
```

When re-loading the draft back into the stores, I will populate `createRulesStep` in the state model with values from I from the `cooperative` property of the filing.

Reply:
Sound good, thanks!

PS is rules file name and size enough to re-populate the rules step when reloading a draft Coop IA?

Reply:
updated using approach mentioned in my previous comment.

Reply:
I'm good with the changes here. Up to Cam or Vysakh whether to resolve this conversation.